pkg/types/plugin.go

```diff
@@ -209,7 +211,13 @@ func (d *pluginDiffer) createUpdatePlugin(plugin *state.Plugin) (*crud.Event, er
 		return nil, fmt.Errorf("failed clearing unmatching deprecations fields: %w", err)
 	}
 
-	if !currentPlugin.EqualWithOpts(pluginWithDefaults, false, true, false) {
+	jsonb, err := json.Marshal(&schema)
+	if err != nil {
```

Comment:
In case of an error, should we consider proceeding to comparison with nil schema object? Error here doesn't seem significant enough to stop the entire process.